trustly-android-sdk/build.gradle

```diff
@@ -52,10 +61,12 @@ task javadoc(type: Javadoc) {
     exclude '**/TrustlyPanel.java'
 }
 
+apply from: 'publish-root.gradle'
+
 ext {
     PUBLISH_GROUP_ID = 'io.github.TrustlyInc'
     PUBLISH_VERSION = '2.2.1'
```

Comment:
Is this version still 2.2.1? I thought this repo was only starting in 3.0.0?

Reply:
In this PR yet! There's another PR just to change the version code